src/components/organisms/proposal-review.tsx

```diff
@@ -49,13 +54,33 @@ export const ProposalReview = (props: {
 
 function renderAction(action: ProposalAction) {
   switch (action.type) {
+    case 'send':
+      return <SendReview values={action.values as ProposalSendFormValues} />
     case 'stake':
       return <StakeReview values={action.values as ProposalStakeFormValues} />
     default:
       return null
   }
 }
 
+const SendReview = ({ values }: { values: ProposalSendFormValues }) => {
+  const { fee } = useSnapshot(Chain)
+  return (
+    <FormCard title="Send">
+      <Stack spacing={SPACING.formStack}>
+        <ReviewItem label="Type">{values.sendType}</ReviewItem>
+        {/*TODO: how to best pass down from address...?*/}
+        {'toAddress' in values && (
+          <ReviewItem label="To Address">{values.toAddress}</ReviewItem>
+        )}
+        {/*TODO: how to best pass down denom...?*/}
```

Comment:
@haveanicedavid any thoughts here?

Reply:
Maybe not the most elegant solution but we can simply pass down group policy address.

Reply:
For denom, we can maybe punt on the displaying something other than regen and tackle as part of #19

Reply:
@ryanchristo I don't think it's necessarily bad to pass through props, but can get a bit tedious over time so I tend to avoid prop drilling where possible. Off the top of my head, I think it makes sense here to include `groupPolicyAddress` and `denom` in the actual form values, even if they remain hidden in the form UI. That way we can avoid prop drilling for extra values, which can get a bit tedious

So for example in `send-single-form`, you'd add `fromAddress` (representing the group policy address) to the form values and require the form be instantiated with that value, but not actually allow it to be modified by the user. 

I think it'll be easier to show in code... i'll make. PR into this shortly to show what I'm thinking

Reply:
actually, ignore the above comment. Passing through props should be fine for now - if we need to convert to something more elegant in the future, we can. 

Also after glancing through things, my suggested approach of adding to form values might not be ideal. Could use a react `context` (or my preference: a jotai atom) in the `proposal-crud-template` which any children could consume, but I don't think that's necessary at this point - how you have it here is fine 👍

Reply:
Ok, sounds good to me. Thanks for the info! I think context makes sense if/when we need to pass more props.